core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -653,7 +653,7 @@ private[spark] class SparkSubmit extends Logging {
       if (opt.value != null &&
           (deployMode & opt.deployMode) != 0 &&
           (clusterManager & opt.clusterManager) != 0) {
-        if (opt.clOption != null) { childArgs += (opt.clOption, opt.value) }
+        if (opt.clOption != null) { childArgs += opt.clOption += opt.value }
```

Comment:
We already make `core` module to pass Scala 2.13 compilation three days ago. Is this required?
- https://github.com/apache/spark/commit/3ad4863673fc46080dda963be3055a3e554cfbc7

cc @srowen